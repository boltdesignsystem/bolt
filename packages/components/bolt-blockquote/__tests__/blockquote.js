import { render, stopServer, renderWC } from '../../../testing/testing-helpers';
import schema from '../blockquote.schema';
const componentSelector = 'bolt-blockquote';
const { size, alignItems, border } = schema.properties;
const languages = ['en', 'de', 'fr', 'ja'];
let page, fixtures;

afterAll(async () => {
  await stopServer();
  await page.close();
}, 100);

beforeEach(async () => {
  await page.evaluate(() => {
    document.body.innerHTML = '';
  });
  await page.setViewport({ width: 800, height: 400 });
});

beforeAll(async () => {
  page = await global.__BROWSER__.newPage();
  await page.goto('http://127.0.0.1:4444/', {
    timeout: 0,
  });

  const defaultData = {
    author: {
      name: 'Michelangelo di Lodovico Buonarroti Simoni',
      title: 'Renaissance Artist',
    },
    content:
      '<p>The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.</p>',
  };

  fixtures = {
    defaultData,
  };
});

describe('Bolt Blockquote', () => {
  test(`default`, async () => {
    const results = await render(
      '@bolt-components-blockquote/blockquote.twig',
      {
        ...fixtures.defaultData,
      },
    );

    await expect(results.ok).toBe(true);
    await expect(results.html).toMatchSnapshot();
  });

  test(`default with web component`, async () => {
    const results = await render(
      '@bolt-components-blockquote/blockquote.twig',
      {
        ...fixtures.defaultData,
      },
    );

    const { innerHTML, outerHTML } = await renderWC(
      componentSelector,
      results.html,
      page,
    );

    await expect(results.ok).toBe(true);
    await expect(results.html).toMatchSnapshot();
    await expect(innerHTML).toMatchSnapshot();
    await expect(outerHTML).toMatchSnapshot();
  });
});

describe('Bolt Blockquote Props', () => {
  alignItems.enum.forEach(async option => {
    test(`align items: ${option}`, async () => {
      const results = await render(
        '@bolt-components-blockquote/blockquote.twig',
        {
          ...fixtures.defaultData,
          alignItems: option,
        },
      );

      await expect(results.ok).toBe(true);
      await expect(results.html).toMatchSnapshot();
    });
  });

  size.enum.forEach(async option => {
    test(`size: ${option}`, async () => {
      const results = await render(
        '@bolt-components-blockquote/blockquote.twig',
        {
          ...fixtures.defaultData,
          size: option,
        },
      );

      await expect(results.ok).toBe(true);
      await expect(results.html).toMatchSnapshot();
    });
  });

  border.enum.forEach(async option => {
    test(`border: ${option}`, async () => {
      const results = await render(
        '@bolt-components-blockquote/blockquote.twig',
        {
          ...fixtures.defaultData,
          border: option,
        },
      );

      await expect(results.ok).toBe(true);
      await expect(results.html).toMatchSnapshot();
    });
  });
});

describe('Bolt Blockquote Language', () => {
  languages.forEach(async option => {
    test(`language: ${option}`, async () => {
      const results = await render(
        '@bolt-components-blockquote/blockquote.twig',
        {
          ...fixtures.defaultData,
          lang: option,
        },
      );

      await expect(results.ok).toBe(true);
      await expect(results.html).toMatchSnapshot();
    });
  });
});
