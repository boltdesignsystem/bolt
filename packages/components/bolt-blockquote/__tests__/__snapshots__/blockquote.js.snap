// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bolt Blockquote Language language: de 1`] = `
<bolt-blockquote lang="de">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Language language: en 1`] = `
<bolt-blockquote lang="en">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Language language: fr 1`] = `
<bolt-blockquote lang="fr">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Language language: ja 1`] = `
<bolt-blockquote lang="ja">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props align items: center 1`] = `
<bolt-blockquote align-items="center">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-center c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props align items: left 1`] = `
<bolt-blockquote align-items="left">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props align items: right 1`] = `
<bolt-blockquote align-items="right">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-end c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props border: horizontal 1`] = `
<bolt-blockquote border="horizontal">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-horizontal">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props border: none 1`] = `
<bolt-blockquote border="none">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--borderless">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props border: vertical 1`] = `
<bolt-blockquote border="vertical">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props size: large 1`] = `
<bolt-blockquote size="large">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--large c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props size: xlarge 1`] = `
<bolt-blockquote size="xlarge">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote Props size: xxlarge 1`] = `
<bolt-blockquote size="xxlarge">
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xxlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote default 1`] = `
<bolt-blockquote>
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote default with web component 1`] = `
<bolt-blockquote>
  <blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
    <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
      <ssr-keep for="bolt-blockquote">
        <p>
          The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
        </p>
      </ssr-keep>
    </div>
    <footer class="c-bolt-blockquote__footer">
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-name"
              class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
        >
          Michelangelo di Lodovico Buonarroti Simoni
        </cite>
      </ssr-keep>
      <ssr-keep for="bolt-blockquote"
                class="c-bolt-blockquote__footer-item"
      >
        <cite slot="author-title"
              class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
        >
          Renaissance Artist
        </cite>
      </ssr-keep>
    </footer>
  </blockquote>
</bolt-blockquote>
`;

exports[`Bolt Blockquote default with web component 2`] = `
<blockquote class="c-bolt-blockquote c-bolt-blockquote--align-items-start c-bolt-blockquote--bordered-vertical">
  <div class="c-bolt-blockquote__quote c-bolt-blockquote__quote--xlarge c-bolt-blockquote__quote--semibold">
    <slot>
    </slot>
  </div>
  <footer class="c-bolt-blockquote__footer">
    <div class="c-bolt-blockquote__footer-item">
      <cite class="c-bolt-blockquote__author-name">
        <slot name="author-name">
        </slot>
      </cite>
    </div>
    <div class="c-bolt-blockquote__footer-item">
      <cite class="c-bolt-blockquote__author-title">
        <slot name="author-title">
        </slot>
      </cite>
    </div>
  </footer>
</blockquote>
`;

exports[`Bolt Blockquote default with web component 3`] = `
<bolt-blockquote>
  <p>
    The greater danger for most of us lies not in setting our aim too high and falling short; but in setting our aim too low, and achieving our mark.
  </p>
  <cite slot="author-name"
        class="c-bolt-text c-bolt-text--bold c-bolt-text--normal c-bolt-text--xsmall"
  >
    Michelangelo di Lodovico Buonarroti Simoni
  </cite>
  <cite slot="author-title"
        class="c-bolt-text c-bolt-text--regular c-bolt-text--normal c-bolt-text--xsmall"
  >
    Renaissance Artist
  </cite>
</bolt-blockquote>
`;
